dotcom-rendering/scripts/gen-stories/get-stories.mjs

```diff
@@ -129,11 +129,14 @@ export const ${storyVariableName} = () => {
 };
 ${storyVariableName}.storyName = '${renderingTarget}: Display: ${displayName}, Design: ${designName}, Theme: ${theme}';
 ${storyVariableName}.args = { config: ${JSON.stringify(config)} };
-${storyVariableName}.decorators = [lightDecorator({
-	display: ArticleDisplay.${displayName},
-	design: ArticleDesign.${designName},
-	theme: {...ArticleSpecial, ...Pillar}.${theme.replace('Pillar', '')},
-})];
+${storyVariableName}.decorators = [splitTheme([
+		{
+			display:  ArticleDisplay.${displayName},
+			design: ArticleDesign.${designName},
+			theme: {...ArticleSpecial, ...Pillar}.${theme.replace('Pillar', '')},
+		},
+	]),
```

Comment:
I think that we want these stories to stack? Or maybe even be two separate screenshots?

```suggestion
	], { orientation: 'vertical' }),
```